src/epsilon/analyzer.d

```diff
@@ -918,15 +923,18 @@ void Analyse(Input input)
     if (ErrorCounter == 0)
     {
         EAG.History |= EAG.analysed;
-        info!"%s grammar is valid"(EAG.BaseName);
+        if (!reportOffset)
```

Comment:
This is more than what you would guess "report offset" to do.
Is this necessary? Then:
shouldn't we have an option like "lsp" that also suppresses all logging below `warn`?

Reply:
Yes, you are right. But was a later decision on my side to skip this last concluded information.
It would help definitely help to get no logging below `warn`. 
And the `--lsp` option looks much nicer! I was already played with that idea too. But the PR was a first try and I knew you would review it. :smiley:

I can do the --lsp option. But what would need to be done the filter out all logging below `warn`? I did not saw a central point in the code to do this...

Reply:
Setting the log levels is as simple as this:
https://github.com/linkrope/gamma/blob/master/src/epsilon/main.d#L62-L63

Reply:
You mean, setting the log level for --lsp option would be similar to:
```
        if (lsp)
                    levels |= Level.warn;
```

Reply:
```
if (lsp)
    levels = Level.error | Level.warn;
```